qiskit_experiments/library/quantum_volume/qv_experiment.py

```diff
@@ -74,29 +73,29 @@ def __init__(
         self,
         qubits: Sequence[int],
         backend: Optional[Backend] = None,
-        trials: Optional[int] = 100,
```

Comment:
Can you  double check these default values are captured in the default experiment options?

Reply:
I have confirmed this, yes.